gnd/build.gradle

```diff
@@ -43,7 +47,7 @@ if (secretsFile.exists()) {
 def googleMapsApiKey = secrets.getProperty('GOOGLE_MAPS_API_KEY', '')
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 30
```

Comment:
Is this change required for the plugin? If not, lets revert this.

Reply:
Reverted